mount_test.go

```diff
@@ -13,9 +13,9 @@ func TestStripErrPathPrefix(t *testing.T) {
 	someError := errors.New("some error")
 	for _, tc := range []struct {
 		err          error
+		expectErr    error
 		name         string
 		mountSubPath string
-		expectErr    error
```

Comment:
Same as fstest, if feasible let's avoid reordering the table driven test case types for better readability.